pkg/controllers/csi/driver/volumes/app/publisher.go

```diff
@@ -80,83 +81,76 @@ func (publisher *AppVolumePublisher) PublishVolume(ctx context.Context, volumeCf
 		return nil, err
 	}
 
-	agentsVersionsMetric.WithLabelValues(bindCfg.MetricVersionLabel()).Inc()
+	agentsVersionsMetric.WithLabelValues(bindCfg.Version).Inc()
 
 	return &csi.NodePublishVolumeResponse{}, nil
 }
 
 func (publisher *AppVolumePublisher) UnpublishVolume(ctx context.Context, volumeInfo *csivolumes.VolumeInfo) (*csi.NodeUnpublishVolumeResponse, error) {
-	volume, err := publisher.loadVolume(ctx, volumeInfo.VolumeID)
+	appMount, err := publisher.db.ReadAppMount(ctx, metadata.AppMount{VolumeMetaID: volumeInfo.VolumeID})
+
 	if err != nil {
 		log.Info("failed to load volume info", "error", err.Error())
 	}
 
-	if volume == nil {
+	if appMount == nil {
 		return &csi.NodeUnpublishVolumeResponse{}, nil
 	}
 
-	log.Info("loaded volume info", "id", volume.VolumeID, "pod name", volume.PodName, "version", volume.Version, "dynakube", volume.TenantUUID)
```

Comment:
why do we want to not log `"dynakube", volume.TenantUUID` ?

Reply:
Because `AppMount` doesn't contain `TenantUUID` anymore.
I missed to rename the log though, fixing that :+1:

Reply:
Done